.github/workflows/ci.yml

```diff
@@ -79,27 +73,20 @@ jobs:
           go-version-file: 'go.mod'
           cache: true
 
-      - name: Install Kubernetes tools
-        uses: ./.github/actions/install-kubernetes-tools
-        with:
-          install-kubectl: true
-          install-kuttl: true
-          install-k3d: true
-
-      - name: Create K3D cluster
-        run: |
-          sudo modprobe br_netfilter
-          make k3d-cluster-up
-
-      - name: Run features tests
+      - name: Check default main state - generated files
         run: |
-          make test-integration-features
+          make release
+          #          git diff --exit-code $(git ls-files | grep -v "bundle/manifests/everest-operator.clusterserviceversion.yaml")
```

Comment:
Leftover?

Reply:
I like the comment option, it makes both accepting and ignoring the change a very easy action for the dev :+1:

Reply:
but now we achieve the same.
in case `make release` creates some changes (diff) - the next step
```
      # post diff as a suggestion to commit to the PR (if any)
      - uses: ./.github/actions/reviewdog-suggester
```
will post them as PR commit suggestion. And developer may decided what to do with it